pkg/tasks/tasks.go

```diff
@@ -104,7 +104,15 @@ func WithHostnameOSAndProbes(t Tasks) Tasks {
 // WithFullInstall with install binaries (using WithBinariesOnly) and
 // orchestrate complete cluster init
 func WithFullInstall(t Tasks) Tasks {
-	return WithBinariesOnly(t).
+	return Tasks{
+		{
+			Fn: func(s *state.State) error {
+				s.Logger.Infoln("Disable nm-cloud-setup nodes...")
+				return s.RunTaskOnLeader(disableNMCloudSetup)
```

Comment:
why only on a leader? aren't other nodes need this to be disabled?

Reply:
Oh, I see why, because of the abuse of this and iterating over all nodes inside the OnLeader task....

Well, we have `s.RunTaskOnAllNodes()` for this

Reply:
But the current node object that we receive on the task doesn't have a property which tells you this node has been initialized or not. And this is quite important.

Reply:
initialized status can be found by ID in `(*state.State).LiveCluster`, just like you did in `disableNMCloudSetup`.

Reply:
This should be s.RunTaskOnAllNodes(), otherwise the service will be disabled only on the leader (first) node.